### PR TITLE
chore/update etl to match qa-covid

### PR DESCRIPTION
**Pull request by gkuffel**

### CODE REVIEW

File: chicagoland.pandemicresponsecommons.org/etlMapping.yaml [hunk 1]
```diff
@@ -15,37 +15,75 @@ mappings:
...
+          - name: state_total_covid_pui_in_icu
+          - name: state_total_covid_pui_in_hospital
+          - name: state_pct_beds_covid_pui
+          - name: testing
+          - name: total_in_use_beds_non_covid
+          - name: total_in_use_beds_covid
+
         sorted_by: updated_datetime, desc
     parent_props:
       - path: projects[project_code:code]
```
Comment (thanh-nguyen-dang) [inline chicagoland.pandemicresponsecommons.org/etlMapping.yaml:89]: line 89 to 92 in new branch should be changed to match with QA etl-mapping, similar to line 131-136
We have this format in QA
projects[project_code:code,project_name:name,project_url:url,project_investigator_affiliation:investigator_affiliation]
projects[project_code:code,project_name:name,project_url:url,project_data_description:data_description,project_investigator_affiliation:investigator_affiliation,project_availability_type:availability_type]

Comment (gkuffel) [reply chicagoland.pandemicresponsecommons.org/etlMapping.yaml:89]: Yes, I did notice that the format is different in QA, but thought it was a bit cleaner shown this way. The code is exactly the same, just different format.

Comment (paulineribeyre) [reply chicagoland.pandemicresponsecommons.org/etlMapping.yaml:89]: i agree it looks cleaner, but we want the prod config file identical to the QA one

Comment (gkuffel) [reply chicagoland.pandemicresponsecommons.org/etlMapping.yaml]: I updated according to the feedback.

**Pull request closed without merging**